CHANGELOG.md

```diff
@@ -10,34 +10,66 @@ Released: TBD
 
 ### Major Changes
 
-- Add support for generating source maps.
-  [@Mingun](https://github.com/peggyjs/peggy/pull/163)
-
-- Introduce an API for reporting errors, warnings and information messages from
-  passes. New API allows reporting several diagnostics at once with intermediate
-  results checking after each compilation stage.
-  [@Mingun](https://github.com/peggyjs/peggy/pull/160)
-
-- Remove support for Node.js version 10.  When updating dependencies, too many
-  of the tools we use no longer work on the Node 10, which went out of
-  security maintenance more than a year ago.  Added support for Node.js
-  version 18.
-  [@hildjj](https://github.com/peggyjs/peggy/pull/248)
+- [#163](https://github.com/peggyjs/peggy/pull/163): Add support for
```

Comment:
Totally supporting moving issue number to the start. If you a not happy with the length of the string, it is possible to use non-inline links:
```md
- [#163]: Add support for
...
[#163]: https://github.com/peggyjs/peggy/pull/163
```

Reply:
I'd just as soon keep them next to each other so I can check that they match, and that I haven't forgotten one.